frame/contracts/src/weights.rs

```diff
@@ -122,1780 +122,1855 @@ pub trait WeightInfo {
 		//  Estimated: `441 + k * (70 ±0)`
-		// Minimum execution time: 13_096_000 picoseconds.
-		Weight::from_parts(13_395_000, 441)
-			// Standard Error: 1_046
-			.saturating_add(Weight::from_parts(1_246_238, 0).saturating_mul(k.into()))
+		// Minimum execution time: 13_291_000 picoseconds.
+		Weight::from_parts(13_825_000, 441)
+			// Standard Error: 1_137
+			.saturating_add(Weight::from_parts(1_244_309, 0).saturating_mul(k.into()))
 			.saturating_add(T::DbWeight::get().reads(2_u64))
 			.saturating_add(T::DbWeight::get().reads((1_u64).saturating_mul(k.into())))
 			.saturating_add(T::DbWeight::get().writes(2_u64))
 			.saturating_add(T::DbWeight::get().writes((1_u64).saturating_mul(k.into())))
 			.saturating_add(Weight::from_parts(0, 70).saturating_mul(k.into()))
 	}
-	/// Storage: unknown `0x4342193e496fab7ec59d615ed0dc553022fca90611ba8b7942f8bdb3b97f6580` (r:2 w:1)
-	/// Proof Skipped: unknown `0x4342193e496fab7ec59d615ed0dc553022fca90611ba8b7942f8bdb3b97f6580` (r:2 w:1)
+	/// Storage: UNKNOWN KEY `0x4342193e496fab7ec59d615ed0dc553022fca90611ba8b7942f8bdb3b97f6580` (r:2 w:1)
+	/// Proof: UNKNOWN KEY `0x4342193e496fab7ec59d615ed0dc553022fca90611ba8b7942f8bdb3b97f6580` (r:2 w:1)
 	/// The range of component `c` is `[0, 125952]`.
 	fn v9_migration_step(c: u32, ) -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `211 + c * (1 ±0)`
 		//  Estimated: `6149 + c * (1 ±0)`
-		// Minimum execution time: 8_409_000 picoseconds.
-		Weight::from_parts(9_006_438, 6149)
+		// Minimum execution time: 8_359_000 picoseconds.
+		Weight::from_parts(9_179_121, 6149)
 			// Standard Error: 1
-			.saturating_add(Weight::from_parts(1_345, 0).saturating_mul(c.into()))
+			.saturating_add(Weight::from_parts(1_312, 0).saturating_mul(c.into()))
 			.saturating_add(T::DbWeight::get().reads(2_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 			.saturating_add(Weight::from_parts(0, 1).saturating_mul(c.into()))
 	}
-	/// Storage: Contracts ContractInfoOf (r:2 w:1)
-	/// Proof: Contracts ContractInfoOf (max_values: None, max_size: Some(290), added: 2765, mode: Measured)
-	/// Storage: System Account (r:1 w:0)
-	/// Proof: System Account (max_values: None, max_size: Some(128), added: 2603, mode: Measured)
+	/// Storage: `Contracts::ContractInfoOf` (r:3 w:1)
+	/// Proof: `Contracts::ContractInfoOf` (`max_values`: None, `max_size`: Some(1819), added: 4294, mode: `Measured`)
+	/// Storage: `System::Account` (r:1 w:0)
+	/// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `Measured`)
 	fn v10_migration_step() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `510`
-		//  Estimated: `6450`
-		// Minimum execution time: 16_962_000 picoseconds.
-		Weight::from_parts(17_716_000, 6450)
-			.saturating_add(T::DbWeight::get().reads(3_u64))
+		//  Measured:  `709`
+		//  Estimated: `9124`
+		// Minimum execution time: 42_457_000 picoseconds.
+		Weight::from_parts(44_556_000, 9124)
```

Comment:
This one increased markedly, on both dimensions. Any ideas why?

Reply:
We did that when you were off https://github.com/paritytech/substrate/pull/14523 but did not update the bench

Reply:
hm but that one changed v12 migration, while the weights change in question is for the v10 one.

Reply:
ah right 🤔